packages/astro/src/default/components/ResizablePanel.astro

```diff
@@ -111,6 +125,18 @@ if (!hasSidePanel) {
       }
     }
 
+    mainPanel(): HTMLElement {
+      return this.#container.children[1] as HTMLElement;
```

Comment:
It feels a bit tricky to rely on the position? Can't we use a data-element or something and grab the correct child?

Reply:
Oh yeah, I'll update it :+1: 

(I did it this way only because the template is above so it's easy to check if it's correct. But if this code ever grows then it won't be as maintainable that `data-id`s)